.eslintrc.json

```diff
@@ -1,22 +1,27 @@
 {
+  "settings": {
```

Comment:
We'll want to back port these changes into the eslint config package as much. But that'll be handled in a later ticket.